week2/index.js

```diff
@@ -0,0 +1,157 @@
+"use-strict";
+
+const fs = require('fs');
+const FILENAMES = { help: "help.txt", todo: "todo.json" }
+
+let readFiles = (filename) => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filename, "utf8", (err, data) => {
+            if (err) {
+                if (err.code === "ENOENT") {
+                    console.log(`Sorry, this file doesn't exist, but new file has been created : ${filename}`);
+                    let fileContent = [];
+                    let filePath = `${filename}.json`;

```

Comment:
Since `filename` is `todos.json`, the file you're reading + writing from is called `files.json.json`.